### PR TITLE
Add related links section to Coronavirus hub pages

### PR DESCRIPTION
**Pull request by gclssvglx**
In order to help users navigate between Coronavirus hub pages, we want to add links to all other Coronavirus hub pages on each Coronavirus hub page within a new UI component.

For example: on the `business` hub page we'd add links to the `education` and `worker` hub pages, and on the `education` hub page we'd add links to the `business` and `worker` hub pages.

This change adds a partial to render these links on the hub pages and adds this partial to the hub page view.

[Trello](https://trello.com/c/m5y819Z0/337-build-the-related-links-element-for-hub-pages)

## Before

screencapture-gov-uk-coronavirus-business-support-2021-04-21-15_30_48

## After

screencapture-collections-dev-gov-uk-coronavirus-business-support-2021-04-21-15_31_28

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+              module: "gem-track-click",
+              track_category: tracking_category,
+              track_action: tracking_action,
+              track_label: link["url"],
+            }
+          end
+        %>
+
+        <%= link_to(
+          link["label"].html_safe,
```
Comment (kevindew) [inline app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb:30]: Are we expecting HTML to be present here? It doesn't seem too safe to do this and the screenshots don't look like we'd need it.

**Review by kevindew — COMMENTED** [review]
Looks good, commit structure is nice 👍 

I've flagged a couple of things for further discussion.

File: app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<%
+  tracking_category ||= nil
+  tracking_action ||= nil
+  tracking = true if tracking_category.present? && tracking_action.present?
+%>
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb:5]: I'll check and remove if there's no tracking required.  Thanks.

File: app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb (hunk 1 above)
Comment (gclssvglx) [reply app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb:30]: Good spot - will remove.  Thanks

File: app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (gclssvglx) [reply app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb]: Yes, tracking is required for these links.  The Trello card has been updated with the tracking details and the code update.

File: app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    padding: true,
+    border_top: 2,
+    margin_bottom: 6
+  } %>
+
+  <ul class="govuk-list">
+    <% related_topic_section["links"].each do | link | %>
+      <li class="covid__topic-list-item">
+        <%
+          data = if tracking
```
Comment (kevindew) [inline app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb:13]: You seem to have an unnecessary conditional here (that perhaps errors, I assume tracking isn't defined?)

Comment (gclssvglx) [reply app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb:13]: Sorry - will fix

**Review by kevindew — APPROVED** [review]

**Pull request merged**